components/utils/checkUserMinistryRole.tsx

```diff
@@ -0,0 +1,18 @@
+import { ministriesNames } from '@/ches/emailConstant';
+
+function checkUserRolesForMinistry(roles: string[]) {
+  return roles.some((role) => role.includes('ministry'));
+}
+
+export default function checkUserMinistryRole(roles: string[]): string | null {
```

Comment:
Let's revise the function to return `array of ministry names` as the user can have privileges over multiple ministries.